services/app-api/src/authn/cognitoAuthn.ts

```diff
@@ -138,7 +139,9 @@ export function userTypeFromAttributes(
             givenName: attributes.given_name,
             familyName: attributes.family_name,
             stateAssignments: [],
-            divisionAssignment: attributes.division,
+            divisionAssignment: isValidCmsDivison(attributes.division)
+                ? attributes.division
+                : undefined,
```

Comment:
do we want undefined or DMCO here?

Reply:
`undefined` is valid here since users can have an unassigned division. If we default that here, any new users will automatically be assigned to `DMCO` later in that code when the user is inserted into the DB.